app/build.gradle

```
@@ -2,14 +2,16 @@ plugins {
     id 'com.android.application'
     id 'kotlin-android'
 }
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-kapt'
 
 android {
     compileSdkVersion 30
     buildToolsVersion "30.0.3"
 
     defaultConfig {
         applicationId "com.example.cocktailmaster"
-        minSdkVersion 16
+        minSdkVersion 21
```

Lên hẳn 24 đi cho máu 
```suggestion
        minSdkVersion 24
```